lib/fluent/plugin/parser_json_in_json.rb

```diff
@@ -1,6 +1,7 @@
 require 'yajl'
+require 'fluent/plugin/parser'
 
-module Fluent
+module Fluent::Plugin
   class TextParser
```

Comment:
`TextParser` namespace is needless.
Could you use `Fluent::Plugin::JSONInJSONParser` instead of `Fluent::Plugin::TextParser::JSONInJSONParser`?